docs/source/reference/evaql/create.rst

```diff
@@ -32,6 +32,22 @@ To register an user-defined function, specify the implementation details of the
     TYPE  Classification
     IMPL  'evadb/udfs/fastrcnn_object_detector.py';
 
+.. _create-udf-train:
+
+CREATE UDF via Training
+----
+
+To register an user-defined function by training a predication model.
+
+.. code-block:: sql
+
+   CREATE UDF IF NOT EXISTS PredictHouseRent FROM
+   (SELECT * FROM HomeRentals)
```

Comment:
I remember you were discussing this. Is `*` working now?

Reply:
It works. It will consider `_row_id` also as an input column. I will fix it in later PRs.